Photon/IoTlib/examples/starter/starter.ino

```diff
@@ -31,6 +31,7 @@ void setup() {
     // You can change the word 'photon' to whatever you like, but it needs to be something
     // that Alexa can recognize. For now, use a single word.
     iot->addBehavior("Led", new Behavior("photon",'>',0, 100));
+    iot->monitorPressure("Room", 30000);
```

Comment:
What do you think about putting this into a separate example instead of the starter example? 
The starter example is intended to be very minimalist, giving folks a place to get started. I'm adding an example for temperature and humidity, so it might be added in that one as well as in the "everything" example.